.github/workflows/helm-lint-test.yml

```
@@ -0,0 +1,64 @@
+        run: |
+          changed=$(ct list-changed)
+          if [[ -n "$changed" ]]; then
+            echo "changed=true" >> $GITHUB_OUTPUT
+            echo "Changed charts: $changed"
+            changed_charts=$(echo "$changed" | tr '\n' ',')
+            changed_charts=${changed_charts%,} # Remove trailing comma
+            echo "changed_charts=$changed_charts" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Add repositories
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2045:warning:1:12: Iterating over ls output is fragile. Use globs

(shellcheck)